lib/webauto/web4cucumber.rb

```diff
@@ -137,20 +137,19 @@ def browser
       elsif @browser_type == :chrome
         logger.info "Launching Chrome"
 
-	      #https://bugs.chromium.org/p/chromium/issues/detail?id=1056073
-	      chrome_caps[:acceptInsecureCerts] = true
-        if Integer === @scroll_strategy
-          chrome_caps[:element_scroll_behavior] = @scroll_strategy
-        end
         if self.class.container?
           chrome_switches.concat %w[--no-sandbox --disable-setuid-sandbox --disable-gpu --disable-infobars --disable-dev-shm-usage]
         end
-        # options = Selenium::WebDriver::Chrome::Options.new
-        # options.add_extension proxy_chrome_ext_file if proxy_chrome_ext_file
-        options = {}
+        options = {browser_name: 'chrome',
+                   accept_insecure_certs: true}
+        options[:element_scroll_behavior] = @scroll_strategy if Integer === @scroll_strategy
         options[:extensions] = [proxy_chrome_ext_file] if proxy_chrome_ext_file
-        url_or_switches = @selenium_url ? {url: @selenium_url} : {switches: chrome_switches}
-        @browser = Watir::Browser.new :chrome, desired_capabilities: chrome_caps, options: options, **url_or_switches
+        if @selenium_url
+          @browser = Watir::Browser.new :chrome, options: options, url: @selenium_url
+        else
+          options[:switches] = chrome_switches
+          @browser = Watir::Browser.new :chrome, options: options
+        end
```

Comment:
Now I see exactly the same error when running my fixes locally 
```
      [03:07:19] INFO> url..
      [03:07:19] INFO> Navigating to: https://console-openshift-console.apps.qe-ui410-1207.qe.devcluster.openshift.com/
      [03:07:23] INFO> Launching Chrome
      invalid argument: cannot parse capability: goog:chromeOptions
      from invalid argument: unrecognized chrome option: elementScrollBehavior (Selenium::WebDriver::Error::InvalidArgumentError)
      0   chromedriver                        0x000000010588cbc9 __gxx_personality_v0 + 573977
      1   chromedriver                        0x00000001058199a3 __gxx_personality_v0 + 102387
      2   chromedriver                        0x00000001053db488 chromedriver + 173192
      3   chromedriver                        0x00000001053f3ffb chromedriver + 274427
      4   chromedriver                        0x00000001053f8c25 chromedriver + 293925

```

Reply:
Looks like this is not a supported option too

Reply:
I can't find this is supported in https://github.com/SeleniumHQ/selenium/blob/trunk/rb/lib/selenium/webdriver/chrome/options.rb#L30

Reply:
We may try to put it into `options["goog:chromeOptions"]` 
```
        options["goog:chromeOptions"] = {}
        options["goog:chromeOptions"][:element_scroll_behavior] = @scroll_strategy if Integer === @scroll_strategy
```

Reply:
with above change together with 
```
client = Watir::HttpClient.new
```

chrome can be launched successfully 
<img width="1528" alt="Screen Shot 2021-12-07 at 11 37 03 AM" src="https://user-images.githubusercontent.com/12692381/144961769-1a46dbea-b1f2-4c08-a9ba-8f9fd9adc485.png">

Reply:
If no changes without this line `client = Watir::HttpClient.new` I will see error 
```
     Net::ReadTimeout with #<TCPSocket:(closed)> (Net::ReadTimeout)
```

Reply:
OK, would you like to make these changes altogether in your work in progress PR or let me update here?

Reply:
@jhou1 It's ok to update here, I can submit Firefox changes in my PR : )

Reply:
@yapei With these configured, I got
```
      [09:48:29] INFO> Launching Chrome
      invalid argument: entry 0 of 'firstMatch' is invalid
      from invalid argument: cannot parse capability: goog:chromeOptions
      from invalid argument: unrecognized chrome option: element_scroll_behavior (Selenium::WebDriver::Error::InvalidArgumentError)
```